Makefile

```diff
@@ -18,18 +18,20 @@ codegen:
 	@npx squid-typeorm-codegen
 
 
-typegenKusama:
+typegen-kusama:
 	@npx squid-substrate-typegen typegenKusama.json
-typegenPolkadot:
+typegen-polkadot:
 	@npx squid-substrate-typegen typegenPolkadot.json
 
+typegen: typegen-kusama typegen-polkadot
 
 up:
 	@docker-compose up -d
 
-
 down:
 	@docker-compose down
 
+deploy: codegen typegen-$(network)
+	echo @API_DEBUG=true npx sqd squid update balances-$(network)@$(version) --source github.com/litentry/squid-balances.git#main -v -e NETWORK=$(network)
```

Comment:
I think we might need to omit the `--source` flag to use local repository code, that way this command should work. I think as it is you'll build the local files but the aquarium won't see them as it'll pull direct from github